server/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "dev": "NODE_ENV=dev nodemon src/server.js",
-    "start": "pm2-runtime start src/server.js"
+    "start": "pm2-runtime start src/server.js --"
```

Comment:
what is this? just curious

Reply:
Allows for additional args to be passed to the server.js script and not be interpreted by pm2